protocol/chat.go

```diff
@@ -275,6 +275,11 @@ func (c *Chat) IsActivePersonalChat() bool {
 	return c.Active && (c.OneToOne() || c.PrivateGroupChat() || c.Public()) && c.CommunityID == ""
 }
 
+func (c *Chat) shouldBeSynced() bool {
+	isPublicChat := !c.Timeline() && !c.ProfileUpdates() && c.Public()
+	return c.Active && (isPublicChat || c.OneToOne() || c.PrivateGroupChat())
```

Comment:
@jrainville the same as in the previous review, we should sync deleted chats as well.